### DISCUSSION TITLE
How to view extraction errors

### DISCUSSION THREAD

**Discussion by sam-cha-tfs** [github/codeql#13883] [General]
Is there a command I can run to view the extraction errors for a scan?

**Comment by smowton** [comment]
You could try `from Diagnostic d select d.getMessage()`, or look at the logs stored in the database directory's `log` subdirectory.

If you want further details, could you say what language you're working in and what diagnostic you want to see?

**Comment by sam-cha-tfs** [comment]
Sorry I was talking about the CLI tool. Basically I am running the CLI on Jenkins and would like a way to output the extraction errors.

**Reply by smowton** [reply]
I'm afraid the CLI doesn't provide an interface to do that; you'll need to examine files in the database directory's `log` subdirectory directly.
